x/dualstaking/keeper/migrations.go

```diff
@@ -375,3 +375,10 @@ func (m Migrator) MigrateVersion3To4(ctx sdk.Context) error {
 	m.keeper.SetDisableDualstakingHook(ctx, false)
 	return nil
 }
+
+// MigrateVersion4To5 sets the MinSelfDelegation param with a default value
+func (m Migrator) MigrateVersion4To5(ctx sdk.Context) error {
+	params := dualstakingtypes.DefaultParams()
```

Comment:
set only the required param, dont overwrite everything

Reply:
dualstaking had no param before that, so I'm using only the required param

Reply:
but you are overwriting all the other params. you should instead:
params := getparams
params.minselfdelegation = types.Defaultparams().minselfdelegation
k.setparams(params)

Reply:
ah ok, gotcha